subsys/emul/espi/emul_espi_host.c

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Emulator for the generic eSPI Host. This supports basic
+ * host operations.
+ */
+
+#define DT_DRV_COMPAT zephyr_espi_emul_espi_host
+
+#define LOG_LEVEL CONFIG_ESPI_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(espi_host);
+
+#include <device.h>
+#include <emul.h>
+#include <drivers/espi.h>
+#include <drivers/espi_emul.h>
+
+struct vw_data {
+	enum espi_vwire_signal sig;
+	uint8_t level;
+	uint8_t dir;
```

Comment:
should this be an enum. It needs a comment as to what it means and valid values

Reply:
I added the comment. I haven't created an enum here to use already defined directions
`
#define ESPI_SLAVE_TO_MASTER     0ul
#define ESPI_MASTER_TO_SLAVE     1ul
`
The similar structure is used in espi_mchp_xec.c (struct xec_signal).
I'm aware that it is not a perfect way, but I wanted to use already created defines.